### DISCUSSION TITLE
Yi 1080P help me

### DISCUSSION THREAD

**Discussion by alteus83** [indomus/forum#49] [Hardware & Firmware]
Salve,

ho letto varie vostre discussioni ma non trovo nessun "firmware hack" per le mie 4 camere comprate su amazon qualche mese fà.

Potete aiutarmi oppure indirizzarmi?

questo è il modello della cam IFUSY44UWSIIJN.........
mentre la versione ora del firmware è 9.0.19.02_202007141512

**Comment by snakuzzo** [comment]
Devi provare a vedere quale va bene...controlla sul sito dei vari progetti...

https://github.com/TheCrypt0/yi-hack-v4
https://github.com/roleoroleo/yi-hack-MStar
https://github.com/roleoroleo/yi-hack-Allwinner

Io sulle YIHome1080 uso Allwinner

**Reply by alteus83** [reply]
ora provo di nuovo, ma il tuo seriale è simile al mio?

**Reply by snakuzzo** [reply]
9FUSY31SGZ......

**Reply by alteus83** [reply]
comunque ora sto flashando allwinner e sembra che stia scrivendo, alcuni dicono che ci vule anche un ora prima che torni blu fisso, sono passati 20 minuti e ancora lampeggia l'arancione, qualcosa sta facendo.
Spero che sia la volta buona, grazie

**Reply by snakuzzo** [reply]
a me ha impiegato nemmeno 5 minuti...e forse esagero pure

**Reply by alteus83** [reply]
comunque nulla, arrivato a 40 minuti ma nessuna luce blu e flash secondo me che non parte perchè la cam comuqnue è sempre con il firmware originale.

**Reply by alteus83** [reply]
forse ho trovato, dovrebbe essere questo 

https://github.com/roleoroleo/yi-hack-Allwinner-v2

come non detto serve farlo da un sistema linux :(

**Reply by snakuzzo** [reply]
e dov'è il problema ?

**Reply by alteus83** [reply]
Scusami, è una settimana che ce l'ho e forse è più semplice di quello che penso.
Come devo fare? Tramite cmd?

**Reply by snakuzzo** [reply]
Scusa eh...ma dove hai letto che serve Linux ?
A me non sembra che [QUI](https://github.com/roleoroleo/yi-hack-Allwinner-v2#installation) ci sia scritto che sia necessario Linux

**Reply by alteus83** [reply]
![image](https://user-images.githubusercontent.com/33469502/112766705-31d8c400-9013-11eb-890f-9eed3d1dc747.jpeg)

Non trovo il file da scaricare da inserire nella SD ma soltanto questa frase nell'immagine tagliata qui sopra.

Se mi puoi aiutare a cercare il link 

Scusami la nabbaggine 😞

**Reply by snakuzzo** [reply]
Ma questo NON C'ENTRA NIENTE !!!
Questo serve per creare un tuo firmware...e non è quello che serve a te!!!!
Sei in confusione secondo me :D

DEVI SEGUIRE IL LINK CHE TI HO PASSATO NEL MESSAGGIO PRECEDENTE "INSTALLATION"

**Reply by alteus83** [reply]
allora.....non riesco atrovare la soluzione o un mio sbaglio che sicuramente c'è.............

Update Procedure
- Format an SD Card as FAT32. It's recommended to format the card in the camera using the camera's native format function. If the card is already formatted, remove all the files. 
- (ho provato anche a formattare dal programma yi home oltre che da desktop ma non camabia nulal secondo me).

- Download the latest release from the Releases page.
- Extract the contents of the archive to the root of your SD card.
  (installo questa y21ga_0.1.5.tar dopo aver unzippato il file)

- Insert the SD Card and reboot the camera
(inseriso la SD e non capisco che intenda lui per riavvio ma credo che intende il riavvio che fà in automatico una volta inserita la SD)

- Wait a minute for the camera to update.
- Check the hack opening the web interface http://IP-CAM:8080
(attendo ma a quell'IP mi dice impossibile raggiungere il sito)

- Don't remove the microSD card (yes this hack requires a dedicated microSD card).
 (ultima cosa perdo anchje un SD con sto procediemnto, credo che passerò alle "ezviz"  )

**Reply by snakuzzo** [reply]
se hai fatto tutto correttamente deve funzionare.
Se non va vuol dire che non è compatibile.
Nel caso apri una issue su github del progetto

**Reply by alteus83** [reply]
grazie della comprensione e del tempo dedicato.

**Reply by snakuzzo** [reply]
Prova a vedere gli altri due hack

**Reply by alteus83** [reply]
Lo sviluppatore mi ha detto di provare con un altra SD, in settimana provo.

**Reply by alteus83** [reply]
Ho risolto il problema cambiando SD, ho usato una verbatim, e ho anche usato 7zip invece che winrar.

Ora devo cercare di abilitare l'entità della videocamera, perché questo hack è veramente completo di cose rispetto a quelli di qualche anno fa.

La config in configurazion.yaml che sto usando è questa ma non mi funziona....

camera:
  - platform: yi
    name: Camera
    host: "192.168.1.100"
    password: my_password_123

Avete consigli?

**Reply by snakuzzo** [reply]
1. Il codice va postato usando il markdown, quindi ti chiedo cortesemente di correggere

2. Prova a dare un'occhiata a [questo](https://github.com/snakuzzo/homeassistant-test/blob/master/packages/yicamera.yaml) mio package

Se lo usi ricordati di abilitare `stream`

**Comment by tempod** [comment]
scusa se riesumo questo vecchi post ma vorrei chiedere se a voi queste videocamere funzionano anche con cavi non originali. ne ho provati 5 diversi ma nessuno funziona?
sapete come mai?

**Reply by snakuzzo** [reply]
Ti dico...proprio oggi ne ho ripristinate 2 e usato cavi non suoi, alimentandole direttamente ddalla USB del PC. Zero problemi
